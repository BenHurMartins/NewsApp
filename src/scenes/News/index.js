import React from 'react';
import {Text} from 'react-native';
import FlatListNews from '../../components/FlatListNews';
import ViewContainer from '../../components/ViewContainer';

const News = () => {
  return (
    <ViewContainer>
      <FlatListNews />
    </ViewContainer>
  );
};

export default News;
