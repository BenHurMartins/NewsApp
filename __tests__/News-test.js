/**
 * @format
 */

import 'react-native';
import React from 'react';
import News from '../src/scenes/News';

// Note: test renderer must be required after react-native.
import renderer from 'react-test-renderer';
describe('Redenring tests for News', () => {
  it('News renders correctly', () => {
    renderer.create(<News />);
  });
  it('News snapshot test', () => {
    const tree = renderer.create(<News />).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
