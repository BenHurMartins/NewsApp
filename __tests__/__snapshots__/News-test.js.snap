// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Redenring tests for News News snapshot test 1`] = `
Array [
  <RCTSafeAreaView
    emulateUnlessSupported={true}
    style={
      Object {
        "backgroundColor": "#65a1dd",
        "flex": 0,
      }
    }
  />,
  <RCTSafeAreaView
    emulateUnlessSupported={true}
    style={
      Array [
        Object {
          "backgroundColor": "#FFFFFF",
          "flex": 1,
        },
        Object {
          "flexDirection": "row",
        },
      ]
    }
  >
    <RCTScrollView
      contentContainerStyle={
        Object {
          "width": 607,
        }
      }
      data={Array []}
      getItem={[Function]}
      getItemCount={[Function]}
      keyExtractor={[Function]}
      onContentSizeChange={[Function]}
      onEndReachedThreshold={0.7}
      onLayout={[Function]}
      onMomentumScrollBegin={[Function]}
      onMomentumScrollEnd={[Function]}
      onScroll={[Function]}
      onScrollBeginDrag={[Function]}
      onScrollEndDrag={[Function]}
      removeClippedSubviews={false}
      renderItem={[Function]}
      scrollEventThrottle={50}
      stickyHeaderIndices={Array []}
      viewabilityConfigCallbackPairs={Array []}
    >
      <View />
    </RCTScrollView>
    <RCTScrollView
      contentContainerStyle={
        Object {
          "alignItems": "center",
          "width": 750,
        }
      }
      data={Array []}
      getItem={[Function]}
      getItemCount={[Function]}
      keyExtractor={[Function]}
      onContentSizeChange={[Function]}
      onEndReached={[Function]}
      onEndReachedThreshold={0.7}
      onLayout={[Function]}
      onMomentumScrollBegin={[Function]}
      onMomentumScrollEnd={[Function]}
      onScroll={[Function]}
      onScrollBeginDrag={[Function]}
      onScrollEndDrag={[Function]}
      removeClippedSubviews={false}
      renderItem={[Function]}
      scrollEventThrottle={50}
      stickyHeaderIndices={Array []}
      viewabilityConfigCallbackPairs={Array []}
    >
      <View />
    </RCTScrollView>
  </RCTSafeAreaView>,
]
`;
